File: 0001_tow_sum/tow_sum.py
from typing import List


class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        for first in range(len(nums)):
            for second in range(first, len(nums)):
                if first < second and nums[first] + nums[second] == target:
                    return [first, second]


if __name__ == '__main__':
    result = Solution()
    print(result.twoSum([3, 2, 3], 6))

File: 0001_tow_sum/tow_sum.go
package main

import "fmt"


func twoSum(nums []int, target int) []int {
	for first := 0; first < len(nums); first++ {
		for second := first + 1; second < len(nums); second++ {
			if nums[first] + nums[second] == target{
				return []int{first, second}
			}
		}
	}
	return nil
}


func main() {
	nums := []int{3, 2, 3}
	t := 6
	var result = twoSum(nums, t)
	fmt.Println(result)
}
